### PR TITLE
Windows fix

### PR DESCRIPTION
**Pull request by robogen**
This pull request resolves an issue where packaging the application contained an error that would break the native modules inherit to itself.

All other things (UI, functionality) are otherwise the same.

This has been tested on Linux and Windows, with both systems showing no issues. @adamfitzgibbon could you take a look at this pull and make sure it works on OSX? After that we should be fine.

### CODE REVIEW

**Comment by adamfitzgibbon** [comment]
Yeah, I might not be able to come in tomorrow, but I'll definitely get that done by Thursday if that's alright.

File: HCCGo/postinstall.js [hunk 1]
```diff
@@ -1,12 +1,42 @@
...
+// nslog fixer
+try {
+    fs.unlinkSync(__dirname + brace + 'node_modules'
+                  + brace + 'nslog' + brace + 'build' + brace 
+                  + 'Release' + brace + 'nslog.node');
+} catch (e) {
+    console.log("nslog.node doesn't exist\n");
+}
+
+const ref = spawn(__dirname + brace + 'node_modules' + brace + '.bin'
```
Comment (djw8605) [inline HCCGo/postinstall.js]: You should use the `Path` module, especially the [path.join](https://nodejs.org/api/path.html#path_path_join_paths) function

File: HCCGo/postinstall.js [hunk 2]
```diff
@@ -1,12 +1,42 @@
...
+const ref = spawn(__dirname + brace + 'node_modules' + brace + '.bin' 
+                  + brace + rebuilder,
+                  ['--version='+target,'--log'],
+		  { cwd: __dirname + brace + 'node_modules' + brace + 'ref',
+		    env: process.env,
+			shell: true });
+console.log('ref output: ' + ref.output);
+console.log('ref error: ' + ref.error);
+
+const ffi = spawn(__dirname + brace + 'node_modules' + brace + '.bin'
```
Comment (djw8605) [inline HCCGo/postinstall.js]: Same, use path.join

File: HCCGo/postinstall.js [hunk 3]
```diff
@@ -1,12 +1,42 @@
...
+    rebuilder = "electron-rebuild.cmd";
+    brace = "\\";
+} else {
+    rebuilder = "electron-rebuild";
+    brace = "/";
+}
+
+// nslog fixer
+try {
+    fs.unlinkSync(__dirname + brace + 'node_modules'
```
Comment (djw8605) [inline HCCGo/postinstall.js]: path.join

**Comment by robogen** [comment]
Latest commit addresses requested changes

**Comment by adamfitzgibbon** [comment]
@robogen It seems to be working when I run grunt package and open it as the app, but when I just grunt run it, everything is still foobar with that wrong module error. Is that expected to happen?

**Comment by adamfitzgibbon** [comment]
Also it seems to take a while to load the work/home directories, so perhaps in the future we should activate a loading gif, because initially I thought it was broken.

**Comment by robogen** [comment]
I think the grunt run issue is because the globally installed version of electron might not be synced up with what we are using for deployment. I made a slight modification to the package.json to install a private version of electron in node_modules and use that, and then we can control the version of that as we need. @adamfitzgibbon mind giving it a shot?

Also, yes, I agree that a loading gif would be helpful.

**Comment by adamfitzgibbon** [comment]
Sorry, missed that you asked me to run it again, seems to work now though.

**Comment by djw8605** [comment]
Great!  Merging.

**Pull request merged**